game-core/src/test/java/games/strategy/triplea/ResourceLoaderTest.java

```diff
@@ -66,35 +67,21 @@ public void testGetMapZipFileCandidates_ShouldIncludeGitHubZipFile() {
   }
 
   private static Matcher<Iterable<File>> containsDirectoryEndingWith(final String name) {
-    return new TypeSafeMatcher<Iterable<File>>() {
-      @Override
-      public void describeTo(final Description description) {
-        description.appendText("iterable containing file with name ").appendValue(name);
-      }
-
-      @Override
-      protected boolean matchesSafely(final Iterable<File> files) {
-        return StreamSupport.stream(files.spliterator(), false)
+    return MatchBuilder.<Iterable<File>>builder()
+        .description("iterable with directionending with: " + name)
```

Comment:
Need a space between `direction` and `ending`.

Reply:
Even more, should be `directory` :)   
Good catch on lack of spacing.

Reply:
7fd2b2c